R/loadReg2.R

```diff
@@ -331,29 +333,46 @@ predictSolute.loadReg2 <- function(
     # errors on this point.
   }
   
+  #deal with mean water/calendar year options
+  #need to filter data down to compelete years, and then send 'total' option to rloadest
+  #data freq, leap years, whatever is in newdata ob
+  if(agg.by %in% c("mean water year", "mean calendar year")) {
+    newdata <- filterCompleteYears(data = newdata, time.step = load.model@fit$time.step, 
+                                   agg.by = agg.by)
+    agg.by = "total" #going to rloadest
+    if(nrow(newdata) > 176000) {
+      stop(paste(strwrap("Sorry, rloadest can't handle more than 176,000 data points, 
+                          and loadflex does not currently support aggregating uncertainties. 
```

Comment:
this line will probably confuse users who don't understand how rloadest and loadflex interact. I think you could change this whole message to:
```
Sorry, rloadest can't aggregate more than 176,000 data points at a time.
Please change the agg.by argument to a shorter period, or else 
supply a shorter period of data for newdata"
```